test/server/config_validation/xds_fuzz.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package test.server.config_validation;
+
+import "validate/validate.proto";
+
+message AddListener {
+  uint32 listener_num = 1 [(validate.rules).uint32.gt = 0];
```

Comment:
nit: won't need a gt > 0 on a uint

Reply:
put this in initially to make the field 'required' by apparently in proto3 fields default to their 0 value so i'll take it out